docs/vercel.json

```diff
@@ -1918,6 +1918,18 @@
       "source": "/integrations/airlift/tutorial/migrate",
       "destination": "/integrations/libraries/airlift/airflow-to-dagster",
       "permanent": false
+    },
+    {
+      "source": "/guides/preview/:path*",
+      "destination": "/labs/:path*"
+    },
+    {
+      "source": "/guides/preview/",
+      "destination": "/labs/"
+    },
+    {
+      "source": "/guides/preview",
+      "destination": "/labs/"
```

Comment:
Not a blocker, but a random thought -

I wonder if there's a way we can structure the docs such that they show up in the "Labs" section, but once they're finalized they can be moved without the URL changing such that we don't need redirects. Might not be worth the hassle.

Reply:
Yeah, that's a good point. I don't think it's worth the hassle right now, plus when we incorporate this content into the rest of the docs, we may need to reorganize it to fit into its new home.